src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -87,14 +95,35 @@ internal object AutoMend : Module(
             }
             paused = false
 
+            if (!shouldMend() && autoDisableComplete) {
+                MessageSendHelper.sendChatMessage("$chatName Mending completed! disabling")
+                disable()
+            }
+
+            if (shouldMend() && pauseAutoArmor && AutoArmor.isEnabled) {
+                AutoArmor.disable()
+                isAutoArmorPaused = true
+            }
```

Comment:
Just do the pause check in auto armor instead of toggling it here

Reply:
what is pause check? thanks

Reply:
Do move this to auto armor instead of toggling it